README.md

```diff
@@ -1 +1,19 @@
-# PlantTracker
\ No newline at end of file
+# PlantTracker
+
+PlantTracker is an application that helps you keep track of your plants. It allows you to record important information about each plant, such as its name, species, watering schedule, and any additional notes you may have.
+
+## Getting Started
+
+### Prerequisites
+
+Before you can use PlantTracker, you'll need to have the following software installed on your machine:
+
+- [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html): The package manager and build tool for Rust.
+- [Node.js](https://nodejs.org/en/download/): A JavaScript runtime environment.
+- [Yarn](https://classic.yarnpkg.com/en/docs/install): A package manager for JavaScript.
+
+If you don't have any of these tools installed, please follow the provided links to install them.
+
+Once you have all the prerequisites installed, you can proceed with the installation and usage of PlantTracker.
+
+### Installation
```

Comment:
Finish the installation instruction.
Include following informations:
- usage of `build-release.py` script
- default IP and port addresses
- information about .env.dev configuration (FRONTEND_STATIC_PATH)

I think that the build-release.py scripti should have the default value of output-path parameter connected to the default value set in .env.dev, to make the default installation process as easy as possible.